velox/docs/monthly-updates/may-2024.rst

```diff
@@ -0,0 +1,65 @@
+***************
+May 2024 Update
+***************
+
+Documentation
+=============
+
+* Publish blog post about `optimizing TRY and TRY_CAST <https://velox-lib.io/blog/optimize-try-more>`_
+
+Core Library
+============
+
+* Optimize TRY and TRY_CAST for cases when many rows fail.
+* Add support for LEFT SEMI FILTER and RIGHT SEMI FILTER merge joins.
+* Add support for DECIMAL input to aggregations over distinct values. :pr:`9850`
+* Fix LEFT merge join with extra filter. :pr:`9862`
+
+Presto Functions
+================
+
+* Add :func:`at_timezone` function.
+* Add support for VARBINARY input to :func:`approx_distinct` aggregate function.
+* Add support for CAST(varchar AS timestamp with time zone).
+* Add support for DECIMAL inputs to :func:`min_by` and :func:`max_by` aggregate functions.
+* Fix :func:`arrays_overlap` function for empty arrays. :pr:`9922`
+* Fix :func:`map_top_n` function to break ties by comparing keys.
+* Fix CAST(IntervalDayTime AS Varchar) for negative intervals. :pr:`9871`
+* Fix :func:`from_base64` for inputs without padding. :pr:`8647`
+
+Spark Functions
+===============
+
+* Add :spark:func:`expm1`, :spark:func:`get`, :spark:func:`rint`, :spark:func:`shuffle`,
+  :spark:func:`soundex`, :func:`unix_seconds`, :spark:func:`width_bucket` functions.
+* Add support for complex type inputs to :spark:func:`hash` and :spark:func:`xxhash64` functions.
+* Fix CAST(tinyint/smallint/integer/bigint as varbinary). :pr:`9819`
+* Fix return type for :spark:func:`sum` aggregate function with REAL input. :pr:`9818`
+
+Performance and Correctness
+===========================
+
+* Add Fuzzers for TableWriter and RowNumber operators.
+* Add custom argument type generators for Presto decimal functions. :pr:`9715`
+* Re-enable testing of merge joins in Join Fuzzer.
```

Comment:
Some works we may also want to mention:
* Enable testing of nested loop join in Join Fuzzer. :pr:\`9901\`
* Add random ordering generation in Window Fuzzer. :pr:\`9258\`

Reply:
Thanks. Added.

Reply:
Can we also add the vectorized merge join PR? 

https://github.com/facebookincubator/velox/pull/9763

Reply:
could also mention that the governance mechanics docs have merged 

https://github.com/facebookincubator/velox/pull/9990

Reply:
Could also mention the improvements to evaluateOnce() helper function:

https://github.com/facebookincubator/velox/pull/9708

Reply:
> Can we also add the vectorized merge join PR?
> 
> #9763

@pedroerp This PR didn't land in May. It will go into June update.

Reply:
Good point! It was only created in May.

Reply:
@mbasmanova governance also landed in early June. The other one could be:

* Improved the flexibility of logical input types in the evaluateOnce() unit test helper method. #9708